knowhere/knowhere/index/vector_index/faiss_common/MetricType.h

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
don't create this new directory "faiss_common", put this file "MetricType.h" under knowhere/utils